End2End/functionannotation/src/main/java/com/microsoft/azure/functions/Blob.java

```diff
@@ -0,0 +1,66 @@
+package com.microsoft.azure.functions;
+
+import com.microsoft.azure.functions.annotation.*;
+
+/**
+ * Azure Functions with Azure Blob.
+ */
+public class Blob {
+    /**
```

Comment:
Instead having these tests under functionannotation folder, can you please combine the tests with [endtoend/Blob.java](https://github.com/JanusWu/azure-functions-java-worker/blob/f84020e0c669e505dd1ede322827b2062da3e8c0/End2End/functest/src/main/java/com/microsoft/azure/functions/endtoend/Blob.java)